src/javascript/detectors/sql_injection/sql_injection.js

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+// {fact rule=sql-injection@v1.0 defects=1}
+var sql = require('mysql')
+var express = require('express')
+var app = express()
+
+var connection = sql.createConnection({
+    host     : 'localhost',
+    user     : 'newUserName',
```

Comment:
nit: can we make this `myUserName` for consistency

Reply:
sure